app/controllers/users.js

```diff
@@ -1,47 +1,105 @@
 import Ember from 'ember';
 import DS from 'ember-data';
 
-const {computed, inject, RSVP} = Ember;
-const {service} = inject;
-const {PromiseArray} = DS;
+const { computed, Controller, inject, isEmpty, RSVP, run } = Ember;
+const { Promise } = RSVP;
+const { service } = inject;
+const { PromiseArray } = DS;
+const { debounce } = run;
+const { trim } = Ember.$;
+
+function cleanQuery(query) {
+  return trim(query).replace(/[\-,?~!@#$%&*+\-'="]/, '');
+}
+
+export default Controller.extend({
+  init() {
+    this._super(...arguments);
+
+    // This needs to be removed after user session re-write
+    this.get('currentUser.model').then((user) => {
```

Comment:
It's a substitute until the `session` service gets implemented. I shouldn't have to send any requests when that happens and this should go away.